### ISSUE TITLE
React to `rsvp yes`

### ISSUE THREAD

**Issue by WesleyAC** [recursecenter/RSVPBot#21]
Being able to react to a RSVPBot message with 👍 to `rsvp yes` could cut down on spam in RSVPBot threads.

**Comment by davidbalbert** [comment]
Ooh, this is a great idea! I don't know how hard it would be to implement. @timabbott does `call_on_each_event` support reactji?

**Comment by timabbott** [comment]
Yep, `call_on_each_event` receives any changes that a Zulip browser does if you register for the right event types.  In this case, you want the "reaction" event type.

**Comment by davidbalbert** [comment]
Great, thanks!

@WesleyAC this seems like this might take a day or two of work. I don't have a ton of time for this now, but I will try to set some aside in the next month or so. If you'd like to give it a shot in the mean time, please go for it! I am happy to help you around the codebase.
